src/app/launcher/import-app/project-summary-importapp-step/project-summary-importapp-step.component.ts

```diff
@@ -142,63 +142,32 @@ export class ProjectSummaryImportappStepComponent extends LauncherStep implement
     );
   }
 
-  /**
-   * Ensure repo name is available for the selected organization
-   */
-  validateRepo(): void {
```

Comment:
Is this function not needed? even to validate ?

Reply:
yes. i have moved to a common place in launcher.component.ts